python_modules/dagster-graphql/dagster_graphql/test/utils.py

```diff
@@ -201,6 +201,8 @@ def infer_pipeline_selector(
         {
             "pipelineName": pipeline_name,
             "solidSelection": op_selection,
+            "assetSelection": [],
```

Comment:
So this now means we have coverage for the case where these are set. Is there a case where these aren't set that we're losing coverage for?

Reply:
Good question. The front end always provides them now, but that's not guaranteed forever and gql api users might not. I'll add coverage

Reply:
updated with a specific case for []. Going to start a discussion about how to handle this going forwards